nsv13/code/datums/weapon_types.dm

```diff
@@ -375,5 +375,23 @@
 	weapon_class = WEAPON_CLASS_LIGHT
 	ai_fire_delay = 20 SECONDS
 
+/datum/ship_weapon/prototype_bsa
+	name = "Prototype Bluespace Artillery"
+	default_projectile_type = /obj/item/projectile/bullet/prototype_bsa
+	burst_size = 1
+	fire_delay = 32 SECONDS
+	range_modifier = 200
+	overmap_firing_sounds = list('nsv13/sound/weapons/bsa_fire.ogg')
+	overmap_select_sound = 'nsv13/sound/weapons/bsa_select.ogg'
+	ai_fire_delay = 32 SECONDS
+	lateral = TRUE
+
+/datum/ship_weapon/burst_torpedoes
```

Comment:
Investigate using fire_torpedo_burst()

Reply:
I... don't know what that is, sorry.

Reply:
Search for the code segment, its related to having AI ships fire their torpedoes in bursts.